guide/language-concepts/bodies.markdown

```diff
@@ -55,6 +55,57 @@ The attributes within the body are then type specific. Bodies of type `perms` co
 
 Such bodies can be reused in multiple promises. Like bundles, bodies can have parameters. The body `mog` also consists of the file permissions, file owner, and file group, but the values of those attributes are passed in as parameters.
 
+#### Body Inheritance
+
+CFEngine 3.8 introduced body inheritance via the `inherit_from`
+attribute. It's a parameterized single-inheritance system, so a body can only inherit from one other body, and it can apply parameters.

```

Comment:
You say single-inheritance, but that might confuse someone so that they think you cannot inherit from a body that inherits from another body.

```cf3
body perms mode_system
{
  mode => "644";
}
body perms system
{
  inherit_from => mode_system;
  owner => "root";
  group => "root"
}
body perms system_inherited
{
  inherit_from => system;
}
```